openshift/templates/react-frontend/bc.yaml

```diff
@@ -18,10 +18,10 @@ objects:
       postCommit: {}
       resources: 
         requests:
-          memory: 500Mi
+          memory: 1Gi
```

Comment:
Should we add more CPU as well?

Also how do we plan to scale it? Did we add any autoscaling logic?

Reply:
We didn't, these are purely the resources used to build the image.  If it ends up needing more resources than we should take the time to figure out why building a react app is requiring so many packages and libraries.